polygon-validator-signer-balance-bot/src/agent.spec.ts

```diff
@@ -0,0 +1,155 @@
+import { FindingType, FindingSeverity, Finding, ethers, HandleBlock, Initialize } from "forta-agent";
+import { provideHandleBlock, provideInitialize } from "./agent";
+import { TestBlockEvent } from "forta-agent-tools/lib/test/index";
+import { createAddress } from "forta-agent-tools/lib/utils";
+import { mockFlag, MINIMUM_THRESHOLD_TO_ETHER } from "./constants";
+
+describe("POLYGON-VALIDATOR-SIGNER BOT TEST SUITE", () => {
+  let handleBlock: HandleBlock;
+  let mockEthersProvider: any;
+  let initialize: Initialize;
+  let flag: mockFlag = { wasOverThresholdAlert: false };
```

Comment:
```suggestion
  let mockFlag: Flag = { wasOverThresholdAlert: false };
```

What Roberto suggested is to change the name of the variable, not that of the type. (The new mock type should thus be removed from `constants.ts`)